scripts/macos_vault_installer.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
cannot stay in vault_installer.sh in the case * in?

Reply:
I thought having separated installers for different OS was clearer

Reply:
for golang yes, for the vault installer if you can conflate in only one it's better imo